chart/env/prod.yaml

```diff
@@ -172,7 +172,36 @@ envVars:
             "stop": ["<|im_end|>"]
           }
         },
-          {
+        {
+          "name": "01-ai/Yi-1.5-34B-Chat",
+          "tokenizer": "01-ai/Yi-1.5-34B-Chat",
+          "description" : "Yi-1.5 is an upgraded version of Yi. It is continuously pre-trained on Yi with a high-quality corpus of 500B tokens and fine-tuned on 3M diverse fine-tuning samples.",
+          "logoUrl": "https://huggingface.co/datasets/huggingchat/models-logo/resolve/main/01-ai-logo.png",
+          "modelUrl": "https://huggingface.co/01-ai/Yi-1.5-34B-Chat",
+          "websiteUrl": "https://www.01.ai",
+          "preprompt": "",
+          "chatPromptTemplate": "{{preprompt}}{{#each messages}}{{#ifUser}}<|im_start|>user\\n{{content}}<|im_end|>\\n<|im_start|>assistant\\n{{/ifUser}}{{#ifAssistant}}{{content}}<|im_end|>{{/ifAssistant}}{{/each}}",
```

Comment:
converted this jinja to handlebars https://huggingface.co/01-ai/Yi-1.5-34B-Chat/blob/main/tokenizer_config.json#L40

Reply:
I removed this, since we can load the chat prompt template from the tokenizer_config.json directly, the model was acting strange with this custom chat template. [(internal discussion)](https://huggingface.slack.com/archives/C055NNJF98Q/p1717399425327839)

Reply:
[commit](https://github.com/huggingface/chat-ui/commit/838eea15dfe5cc0adc1ae1faa7763bd6b1013c1f)